README.md

```diff
@@ -76,12 +76,25 @@ models.
   Kai](https://docs.google.com/presentation/d/1awMdp5hHC6L4Xc_uY6Kj4XiskAArDGPhyQRBI6GJUAo/)
 - See other technical design related information at [docs/design](docs/design)
 
-### 🗣️ Blog Posts
-
-- 2024 August 29: [Incident Storage in Kai - A Deep Dive](https://www.konveyor.io/blog/kai-incident-storage-2024/)
-- 2024 July 23: [Embracing the Future of Application Modernization with KAI](https://shaaf.dev/post/2024-07-23-embracing-the-future-of-app-mod-with-konveyor-ai/)
-- 2024 May 07: [Apply generative AI to app modernization with Konveyor AI](https://developers.redhat.com/articles/2024/05/07/modernize-apps-konveyor-ai)
-- 2024 May 07: [Kai - Generative AI Applied to Application Modernization](https://www.konveyor.io/blog/kai-deep-dive-2024/)
+### 🗣️ Conference Talks and Blog Posts
+
+- 2025 April 01: [Project Lightning Talk: Revolutionizing Legacy Migrations with
+  Konveyor AI - Jonah Sussman](https://kccnceu2025.sched.com/event/1tcwv)
+- 2025 February 25: [Upgrade your code with AI: Build Your Own Amazon
```

Comment:
Do you want to link this twice or just once.
We have it here and below in the "Additional YouTube Videos from Community Members"